popup.js

```diff
@@ -0,0 +1,77 @@
+const popup = (project) => {
+  const overlay = document.createElement('div');
+  overlay.className = 'overlay';
```

Comment:
Please kindly fix the alignment and paddings of the popup in order to look like in the template, make sure you center it as well.

![image](https://github.com/memicelis/my-portfolio/assets/118063058/96d617a1-80b3-42de-b7c6-47815b843726)

As well on the mobile section.

![image](https://github.com/memicelis/my-portfolio/assets/118063058/c39a149f-22b5-4c46-b75f-1b72699bc56d)